flow/Platform.actor.cpp

```diff
@@ -4104,53 +4097,53 @@ void platformSpecificDirectoryOpsTests(const std::string& cwd, int& errors) {
 	ASSERT(symlink("../backups/four", "simfdb/backups/five") == 0);
 
 	errors += testPathFunction2(
-	    "abspath", abspath, "simfdb/backups/four/../two", true, true, joinPath(cwd, "simfdb/backups/one/two"));
+	    "abspath", abspath, "simfdb/backups/four/../two", true, true, cwd / "simfdb/backups/one/two");
 	errors += testPathFunction2(
-	    "abspath", abspath, "simfdb/backups/five/../two", true, true, joinPath(cwd, "simfdb/backups/one/two"));
+	    "abspath", abspath, "simfdb/backups/five/../two", true, true, cwd / "simfdb/backups/one/two");
 	errors += testPathFunction2(
-	    "abspath", abspath, "simfdb/backups/five/../two", true, false, joinPath(cwd, "simfdb/backups/one/two"));
+	    "abspath", abspath, "simfdb/backups/five/../two", true, false, cwd / "simfdb/backups/one/two");
 	errors += testPathFunction2("abspath", abspath, "simfdb/backups/five/../three", true, true, platform_error());
 	errors += testPathFunction2(
-	    "abspath", abspath, "simfdb/backups/five/../three", true, false, joinPath(cwd, "simfdb/backups/one/three"));
+	    "abspath", abspath, "simfdb/backups/five/../three", true, false, cwd / "simfdb/backups/one/three");
 	errors += testPathFunction2("abspath",
 	                            abspath,
 	                            "simfdb/backups/five/../three/../four",
 	                            true,
 	                            false,
```

Comment:
Yep. you just a vector of tuples. and you write the test before and iterate over the fectory and call testPath2Function and collect errors. But it is out of scope for this pr i guess.